modules/upgrade/pages/migrate-3x-4x.adoc

```diff
@@ -24,19 +24,25 @@ We strongly recommend that you contact SUSE Consulting to assist with this proce
 
 == Prepare to Migrate
 
-The source system must be running {productname} 3.2 with all the latest updates applied.
+The source system must be running {productname} 3.2 with all the latest updates applied.
 Before you start, ensure that the system is up to date and all updates have been installed successfully.
 
+[IMPORTANT]
+====
+It is important that PostgreSQL 10 is already running on your {productname} 3.2 system.
+For more information, see xref:db-migration.adoc[].
+====
+
 During migration, the database on the source system needs to get exported, or dumped.
 The dump is compressed, and temporarily stored on the target system.
 The compression is done using [command]``gzip`` using the default compression options.
-Maximum compression only yields about 10% of space savings.
+Maximum compression only yields about 10 % of space savings.
```

Comment:
Chicago (9.18, 16th ed) says no space between the numeral and the % symbol.

Reply:
thanks for looking it up!